logexp/Dockerfile

```diff
@@ -0,0 +1,41 @@
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This file builds an image for the log exporter tool. For more info,
+# have a look at the tool's README.md file.
+
+FROM gcr.io/google-containers/kubekins-test:v20160822
```

Comment:
Done. I was desperately searching for one such small image that has python, thanks a lot.
After trying alpine, python:slim and debian, it seemed most easiest/smallest to make it work on alpine, so sticked to it.